build/Makefile

```diff
@@ -14,6 +14,13 @@ ifndef ENABLE_DEBUG
 $(error "ENABLE_DEBUG is not set.  Please re-run configure")
 endif
 
+# Include the version file
+include ../../docker.version
+
+ifndef CONTAINER_BUILDVERSION_STATUS
+$(error "Is Makefile.version missing?  Please re-run configure")
```

Comment:
Yeah, that's true. This is Makefile.version in other stuff. Here, I just decided to make this fixed (no need to have a temporary one, we only support building with superproject).

I'll fix the comment.